bin/server.ts

```diff
@@ -77,6 +93,40 @@ const httpHandler = new AuthenticatedLdpHandler({
 
 const httpServer = new ExpressHttpServer(httpHandler);
 
-httpServer.listen(port);
+// Set up acl so everything can still be done by default
+// Note that this will need to be adapted to go through all the correct channels later on
+const aclSetup = async(): Promise<void> => {
```

Comment:
Not needed for this PR, but I'd prioritize cleaning this up.

Reply:
Not sure what the best solution is. We need some sort of init script that probably takes some user settings to set up everything the first time. But then also has to check what already exists since the current ResourceStore forgets everything on reset, while most other ResourceStores can check if something already exists or not.

Reply:
Regarding this init script, I think a lot of lessons can be learned from NSS.
Partial re-inits can become complex due to conflicts when users have changed things afterwards, so we have to be sure that these are handled properly.

On the one hand, we can start simple, and just allow a single init for now. On the other hand, we need to start something that we can build upon in the future without breaking things.

Maybe we'll want both an init script, and a migration script. This migration script should be able to apply diffs to go from any old version of init files to the latest version. A requirement for this would be to store the version the init was run at.

@RubenVerborgh probably also has some thoughts on this.

Reply:
Let's just not have inline Turtle, but put it in a file that we read synchronously.

Reply:
@rubensworks But to your full question: we need to set up the dependency injection, and we probably want some kind of configuration manager (command-line based perhaps) like we did for Comunica. And then some one-time generation scripts or so. But I follow what you wrote.

Reply:
Putting it in a turtle file makes it impossible to dynamically insert the the URI of the root container. What would be best here, use some sort of template or just hardcode `localhost:3000` for now? (Was going to use `<>` first but forgot that will actually refer to the acl file and not the root container itself).

Reply:
OK, we can leave it for now.